roles/openshift_master/vars/main.yml

```diff
@@ -7,10 +7,12 @@ openshift_master_policy: "{{ openshift_master_config_dir }}/policy.json"
 scheduler_config:
   kind: Policy
   apiVersion: v1
-  predicates: "{{ openshift_master_scheduler_predicates
+  predicates:
+    - "{{ openshift_master_scheduler_predicates
```

Comment:
These variables should already be lists, is there a case you are seeing where that is not the case?